### PR TITLE
Remove deprecated platforms from plugin page

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected
None

### What does this PR do?
Update plugin list page to remove deprecated platforms.

### What testing has been done on this change?
Manual Testing

### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

File: www/static/plugins/supportedplatforms.jsx [hunk 1]
```diff
@@ -6,74 +6,34 @@ var Preact = require('preact'),
...
-            {present:false, icon: "osx", alt:"OS X"},
-            {present:false, icon: "browser", alt:"Browser"}
-        ];
+        var sortedPlatforms = {
+            'cordova-android': {present:false, icon: "android", alt:"Android"},
+            'cordova-ios':{present:false, icon: "ios", alt:"iOS"},
+            'cordova-windows':{present:false, icon: "windows", alt:"Windows"},
+            'cordova-osx':{present:false, icon: "osx", alt:"OS X"},
+            'cordova-browser':{present:false, icon: "browser", alt:"Browser"}
+        };
```
Comment (raphinesse) [inline www/static/plugins/supportedplatforms.jsx]: Objects so bit guarantee order. So to have something sorted you would have to add an extra stay of keys.

Comment (janpio) [reply www/static/plugins/supportedplatforms.jsx]: Can you take care of this @erisu ? 

Very much needed update to the plugin page.

Comment (erisu) [reply www/static/plugins/supportedplatforms.jsx]: @janpio Sure, I just submitted a new PR with the corrections. 

Some how I lost branch to this PR and cant update this ha. Please check out the new changes on PR #848.

@raphinesse Please also take a look if you have time.

**Comment by erisu** [comment]
Closing this PR and replaced with PR #848. This PR's branch had been lost.

**Comment by janpio** [comment]
@brodybits, @erisu deleted/lost the branch in his fork which is why he couldn't work on it any more ;)

**Pull request closed without merging**